src/ring_theory/unique_factorization_domain.lean

```diff
@@ -439,6 +451,29 @@ end
 lemma dvd_of_mem_factors {p a : α} (h : p ∈ factors a) : p ∣ a :=
 dvd_trans (multiset.dvd_prod h) (associated.dvd (factors_prod (ne_zero_of_mem_factors h)))
 
+lemma factors_mul {x y : α} (hx : x ≠ 0) (hy : y ≠ 0) :
+  multiset.rel associated (factors (x * y)) (factors x + factors y) :=
+begin
+  refine factors_unique irreducible_of_factor
+    (λ a ha, (multiset.mem_add.mp ha).by_cases (irreducible_of_factor _) (irreducible_of_factor _))
+    ((factors_prod (mul_ne_zero hx hy)).trans _),
+  rw multiset.prod_add,
+  exact (associated.mul_mul (factors_prod hx) (factors_prod hy)).symm,
+end
+
+@[simp] lemma factors_pow {x : α} (n : ℕ) :
```

Comment:
It seems a little odd to me that one of these lemmas is simp and the other isn't, right now `pow_two` will affect if simp fires or nor.

Reply:
You're right, I didn't pay attention to `simp`. @Ruben-VandeVelde what do you think?

Reply:
Oh, me neither. `normalized_factors_mul` and `normalized_factors_pow` are both `simp` lemmas, so I'll add `factors_mul` too. Thanks for pointing that out.